README.md

```diff
@@ -1,8 +1,3 @@
 # Chat Terminal
 
 A terminal based chat application for super fast communication. Based on MQTT protocol.
-
```

Comment:
Are you moving this somewhere? like in issues as we discussed or do you want me to put it there?

Reply:
You put it in the issues.